mupen64plus-rsp-paraLLEl/jit_allocator.cpp

```diff
@@ -31,7 +32,12 @@ Allocator::~Allocator()
 
 static size_t align_page(size_t offset)
 {
-	return (offset + 4095) & ~size_t(4095);
+#ifdef _WIN32
+    size_t pagesize = 4095;
+#else
+    size_t pagesize = sysconf(_SC_PAGESIZE) - 1;
```

Comment:
sysconf isn't available on all systems, for libnx i provide it myself from inside RA.
Can't rely on that for the core tho